tests/llms/test_litellm.py

```diff
@@ -0,0 +1,243 @@
+import os
+from typing import Any, AsyncGenerator, Generator
+
+import litellm
```

Comment:
This is also likely causing unit tests to fail

If lite-llm isn't installed, we should probably be skipping the tests

See the tests for other llms as an example (anthropic, palm, etc.)